worker/uniter/uniter_test.go

```diff
@@ -204,6 +206,21 @@ func (s *UniterSuite) TestUniterInstallHook(c *gc.C) {
 	})
 }
 
+func (s *UniterSuite) TestUniterUpdateStatusHook(c *gc.C) {
```

Comment:
It is suficcient, the tests for the additional calls of update status are in the following patch which, additionally has those calls :)